h2o-docs/src/product/getting-data-into-h2o.rst

```diff
@@ -190,52 +192,135 @@ core-site.xml must be configured for at least the following properties (class, p
 
-Requirements:
+Requirements
+''''''''''''
 
-- Direct Metastore access - Hive jars and configuration must be present on H2O job classpath - either via adding it to yarn.application.classpath (or similar property for your resource manger of choice) or by adding Hive jars and configuration to libjars.
-- JDBC metadata access - Hive JDBC Driver must be on H2O job classpath. (See below for details about access Kerberized Hive via JDBC.)
 - The user running H2O must have read access to Hive and the files it manages.
 
+- For Direct Metastore access, the Hive jars and configuration must be present on H2O job classpath - either by adding it to yarn.application.classpath (or similar property for your resource manger of choice) or by adding Hive jars and configuration to libjars. Note that Direct Metastore access is the perferred method.
+
+- For JDBC metadata access, the Hive JDBC Driver must be on H2O job classpath.
+
 Limitations
+'''''''''''
 
-- Imported table must be stored in a format supported by H2O. (See above.)
-- CSV - Hive table property ``skip.header.line.count`` is currently not supported, CSV files with header rows will be imported with header row as data
-- Partitioned tables with different storage formats. H2O supports importing partitioned tables that use different storage formats for different partitions; however in some cases (for example large number of small partitions), H2O may run out of memory while importing even though the final data would easily fit into the memory allocated to the H2O cluster.
+- The imported table must be stored in a :ref:`format supported by H2O<supported_file_formats>`. 
+- CSV: The Hive table property ``skip.header.line.count`` is currently not supported. CSV files with header rows will be imported with the header row as data.
+- Partitioned tables with different storage formats. H2O supports importing partitioned tables that use different storage formats for different partitions; however in some cases (for example large number of small partitions), H2O may run out of memory while importing, even though the final data would easily fit into the memory allocated to the H2O cluster.
+
+Importing Examples
+''''''''''''''''''
+
+Example 1: Access Metadata via Metastore (preferred)
+####################################################
+
+This example shows how to access metadata via the metastore. 
+
+1. Add the Hive JDBC driver to H2O's classpath.
+
+ .. code-block:: bash
+
+      # Add the Hive JDBC driver to H2O's classpath
+      java -cp hive-jdbc.jar:<path_to_h2o_jar> water.H2OApp
```

Comment:
for direct hive import h2o needs to be running on hadoop so this does not apply, only the paragraph 2. below